### DISCUSSION TITLE
Add pet --exec command to shell history

### DISCUSSION THREAD

**Discussion by dgrabla** [knqyf263/pet#272] [Q&A] [answered]
Hi there,

I really like your snippet bookmark manager but there is something that bugs me: When I want to exec a command, I am always presented a list AFAIK there is no shortcut to just rerun the last command selected the same way it was run the last time (using the same variables etc) 

Is there an option or a workaround to re-run the last command? I could not find the option or a 'history' log I could use.

**Comment by RamiAwar** [comment, marked as answer]
What you can do is use pet search --query instead using the shell functions [mentioned in the README here](https://github.com/knqyf263/pet?tab=readme-ov-file#select-snippets-at-the-current-line-like-c-r)

What this will do is copy the command to your shell for you to execute, that way it will get stored in shell history and you can easily find it with arrow key up for example!
